composer.json

```diff
@@ -12,13 +12,12 @@
     ],
     "require": {
         "php": "^5.5 || ^7.0",
-        "api-clients/psr7-oauth1": "dev-master",
+        "api-clients/psr7-oauth1": "^2.0 || ^1.0",
         "php-http/message": "^1.0"
     },
     "require-dev": {
         "phpspec/phpspec": "~2.3"
     },
-    "minimum-stability": "dev",
```

Comment:
I would like to keep this so that we can always run a dedicated job on Travis testing this package with upcoming releases of `api-clients/psr7-oauth1`.